bundles/org.pitest.pitclipse.core/src/org/pitest/pitclipse/core/preferences/PitPreferences.java

```diff
@@ -20,25 +20,39 @@
  * Constants used to deal with Pitclipse's preferences.
  */
 public final class PitPreferences {
+    public static final String MUTATORS_LABEL = "Mutators";
+    public static final String MUTATORS = "pitMutators";
 
-    public static final String PIT_MUTATORS = "pitMutators";
+    public static final String INDIVIDUAL_MUTATORS = "pitIndividualMutators";
```

Comment:
On second read that's actually confusing. The existing `MUTATORS` preference is already used to store individual mutators. By browsing the code it looks like now it sometimes hold the name of the mutator group, and sometimes the raw list of mutators (e.g., in `PitMutatorsTab::performApply`).

I would have rather see a new constant called `MUTATOR_GROUP` which corresponding preference stores the name of the current group (default, stronger, etc. or "custom"). In case the preference is set to "custom" we query the list of individual mutators from `MUTATORS`.

Reply:
Should be resolved, would you mind taking a look at my solution?

Reply:
The changes look good to me, thank you very much